server/game/gamesteps/triggeredabilitywindow.js

```diff
@@ -50,8 +58,16 @@ class TriggeredAbilityWindow extends BaseAbilityWindow {
             if(abilityChoice.text !== 'default') {
                 title += ' - ' + abilityChoice.text;
             }
+
             return { text: title, method: 'chooseAbility', arg: abilityChoice.id, card: abilityChoice.card };
         });
+
+        if(this.isCancellableEvent(player)) {
+            buttons.push({ timer: true, method: 'pass', id: uuid.v1() });
```

Comment:
Not sure I like the idea of pushing fake buttons onto the prompt in order to get the timer. Why not set this as a property at the prompt-level? e.g.
```javascript
activePrompt: {
    menuTitle: 'Any interrupts?',
    buttons: [
        // ...
    ],
    timer: { method: 'pass' }
}
```

Doing it that way, you could also generate the "I need more time" and "Don't ask again until X" buttons client-side as well, giving us more control over their placement / styling.